eos/saveddata/ship.py

```diff
@@ -52,6 +52,9 @@ def __init__(self, item, parent=None):
         self.__itemModifiedAttributes.original = dict(self.item.attributes)
         self.__itemModifiedAttributes.original.update(self.EXTRA_ATTRIBUTES)
         self.__itemModifiedAttributes.overrides = self.item.overrides
+        
+        if "maximumRangeCap" in self.__itemModifiedAttributes.original:
+            cappingAttrKeyCache["maxTargetRange"] = "maximumRangeCap"
```

Comment:
It needs to be either the name or ID of the attribute that holds the new cap value.